distributed/shuffle/tests/test_rechunk.py

```diff
@@ -161,6 +161,7 @@ def test_raise_on_fuse_optimization():
         rechunk(x, chunks=new, method="p2p")
 
 
+@pytest.mark.xfail()
```

Comment:
This is also now no longer expected to raise an error and so the `xfail` is a bit of a misnomer I think?

Reply:
Yup, this is still work in progress; I've adjusted the test.